RELEASE-NOTES.md

```diff
@@ -1,5 +1,52 @@
 # Prefect Release Notes
 
+## Release 2.14.21
+
+### Introducing work queue status
+
+We're excited to unveil the new status indicators for work queues in Prefect's UI, enhancing your ability to oversee and control flow run execution within our hybrid work pools.
+
+Work queues will now display one of three distinct statuses:
+
+- `Ready` -  one or more online workers are actively polling the work queue
+- `Not Ready` - no online workers are polling the work queue, signaling a need for intervention
+- `Paused` - the work queue is intentionally paused, preventing execution
+
+--Add prefect dashboard snapshot here --
+
+--Add work pools page work queues table here with work queues of all statuses.--
+
+With the introduction of work queue status, you'll notice the absence of deprecated work queue health indicators in the UI.
+
+See the documentation on [work queue status](https://docs.prefect.io/latest/concepts/work-pools/#work-queues) for more information.
+
+
+For now, this is an experimental feature, and can be enabled by running:
```

Comment:
Is this true for Prefect Cloud also?

Reply:
@zhen0 the UI feature flag for this will also enable automations to be created for work queue status triggers, which are not yet ready. so this is not available in Cloud as there is more work required to enable that flag.